.github/workflows/static.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - name: PHPStan
-        uses: docker://jakzal/phpqa:latest
+        uses: docker://jakzal/phpqa:php8.0
```

Comment:
Is there any reason to prefer `php8.0` over `latest`?
I don't have any opinion regarding this change, this is just to know yours. :)

Reply:
Yes, the latest may be for the next version of PHP, which would not be accurate.